system/mrm_handler/src/mrm_handler/mrm_handler_core.cpp

```diff
@@ -163,48 +163,38 @@ void MrmHandler::onOperationModeState(
   operation_mode_state_ = msg;
 }
 
-autoware_auto_vehicle_msgs::msg::HazardLightsCommand MrmHandler::createHazardCmdMsg()
+void MrmHandler::publishHazardCmd()
 {
   using autoware_auto_vehicle_msgs::msg::HazardLightsCommand;
   HazardLightsCommand msg;
```

Comment:
A minor point, but it would be better to assign the value of stamp to HazardLightsCommand.
```
msg.stamp = this->now();
```

Reply:
I noticed that in the previous implementation any timestamp was not set to hazard light command topics.
Is it a bug or intended implementation?

Reply:
It looks like a bug!
The vehicle_cmd_gate is passing this TOPIC as is.
In fact, there is no timestamp in hazard_light_cmd when emergency_stop is selected.
The vehicle_cmd_gate is passing this TOPIC as is.

Reply:
Since this it a refactoring Pull Request, let's fix the bug in another PR.